src/main.js

```diff
@@ -105,6 +105,19 @@ class Entry {
     this.defaultValue = defaultValue;
     this.comment = comment;
   }
+
+  html(){
+    return outputHTML([this]);
```

Comment:
We need `Entry.html()` to return a Promise that will provide the output HTML code as an argument to the `resolve` argument.

## Example

```javascript
const entry = new Entry('ADMIN_EMAIL', 'email');

entry.html().then(output => {
  const outputContainer =  document.getElementById("output-container");
  outputContainer.innerHTML = output;
});
```